data-store-api/routes/registry/credentials.py

```diff
@@ -31,6 +31,8 @@ def allows_credential_generation(dataset: ItemDataset) -> bool:
 
     Ensures that the dataset is reposited before enabling credential generation.
 
+    NOTE: RRAPIS-1581 changed this behaviour to enable storage access even if externally reposited.
```

Comment:
⚠️ ⚠️ ⚠️ ⚠️ PLS REMOVE ⚠️ ⚠️ ⚠️ ⚠️  NO RRAP REFERENCE ⚠️ ⚠️ ⚠️ ⚠️

Reply:
Think we'll be okay!